packages/waku/src/lib/plugins/vite-plugin-rsc-analyze.ts

```diff
@@ -45,7 +50,8 @@ export function rscAnalyzePlugin(
           }
         }
       }
-      return code;
+      // TODO this isn't efficient. let's refactor it in the future.
+      return (rscTransform as any).call(this, code, id, options);
```

Comment:
Or can we just do this?
```js
if (clientFileSet.has(id)) {
  return '';
}
```

Reply:
1- This just disables analyzing the tree, basically it'd stop here which is not correct. I believe analyzing should involve analyzing everyfile in the project
2- This might cause errors since it'd mark many files as empty :)

Reply:
1: See the transformed result by `rscTransform`, it'd stop here anyway.
2: How can we avoid causing errors?

Reply:
1- but we are returning something 
2- hmm, I don't have any idea, since if the module exports things but we return nothuling, then it's not something vite expects when analyzing exports. It's a fundemental issue I believe it is if we do this.

Reply:
I don't get the real issue we may have. Let's try and see if it works.

Reply:
Yes, got an error: `Error [RollupError]: "Children" is not exported by "../../packages/waku/dist/client.js", imported by "../../packages/waku/dist/router/defineRouter.js".`

Reply:
So, we need to export default and names (and `*`). That's possible but quite a bit of work.
Okay, I wanted to fix, but let's do it in the future.

Reply:
What I suggest is not to deal that much with user's exports by hacks, that might be too hacky and dangerous.

Reply:
well, the react library does it anyway. so, it's just that who takes the responsibility.

our approach is good for the moment, but the problem is that the build process is getting slower and slower. (maybe caching solves it in this specific case.)